src/Main/Topics.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getTopics } from "../utils/api";
+
+export const Topics = () => {
+  const [topics, setTopics] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getTopics().then((topicsData) => {
+      setTopics(topicsData);
+      setIsLoading(false);
+    });
+  }, []);
+
+  return (
+    <main>
+      {isLoading ? (
+        <p>Loading topics...</p>
+      ) : (
+        <section className="section__topics">
+          <h2>Select a topic</h2>
+          <ul>
+            {topics.map((topic) => {
+              return (
+                <li key={topic.slug}>
+                  <Link to={`/topics/${topic.slug}`}>
+                    <button className="button__topic" value={topic.slug}>
+                      {topic.slug}
+                    </button>
```

Comment:
Semantically this is should just be a `Link`. As a general rule `Links` are already clickable so you don't need the button to highlight that it's intractable as they're technically 2 different elements, one of which does nothing.